### PR TITLE
Fixed #23842 -- Fixed flaky GeoQuerySetTest.test_make_line() test.

### PR DESCRIPTION
**Pull request by felixxm**
ticket-23842

### CODE REVIEW

File: tests/gis_tests/geoapp/tests.py [hunk 1]
```diff
@@ -645,18 +645,15 @@ def test_make_line(self):
...
         # Reference query:
         # SELECT AsText(ST_MakeLine(geoapp_city.point)) FROM geoapp_city;
-        ref_line = GEOSGeometry(
-            "LINESTRING(-95.363151 29.763374,-96.801611 32.782057,"
-            "-97.521157 34.464642,174.783117 -41.315268,-104.609252 38.255001,"
-            "-95.23506 38.971823,-87.650175 41.850385,-123.305196 48.462611)",
-            srid=4326,
-        )
         # We check for equality with a tolerance of 10e-5 which is a lower bound
         # of the precisions of ref_line coordinates
```
Comment (claudep) [inline tests/gis_tests/geoapp/tests.py:649]: I guess the comment may be wiped out. You could however add a new comment below explaining the workaround?

File: tests/gis_tests/geoapp/tests.py [hunk 2]
```diff
@@ -645,18 +645,15 @@ def test_make_line(self):
...
         line = City.objects.aggregate(MakeLine("point"))["point__makeline"]
-        self.assertTrue(
-            ref_line.equals_exact(line, tolerance=10e-5), "%s != %s" % (ref_line, line)
-        )
+        ref_points = City.objects.order_by("point")
+        self.assertEqual(len(line), ref_points.count())
```
Comment (claudep) [inline tests/gis_tests/geoapp/tests.py:652]: Maybe also add an assertion that line is indeed a LINESTRING object?

Comment (felixxm) [reply tests/gis_tests/geoapp/tests.py:653]: Added.

File: tests/gis_tests/geoapp/tests.py (hunk 1 above)
Comment (felixxm) [reply tests/gis_tests/geoapp/tests.py]: Added.

**Review by claudep — APPROVED** [review]
Nice!

**Comment by felixxm** [comment]
@claudep Thanks for checking :+1:

**Pull request merged**